app/views/orders/new.html.slim

```diff
@@ -0,0 +1,27 @@
+- if @event.has_expired?
+  = render partial: "error", locals: { error: "This event is no longer available." }
+- else
+  .container-fluid.tickets-buy
+    = render partial: "header", locals: { event: @event }
+    = form_for [@event, @order] do |f|
+      .row
+        .col-md-11.col-md-offset-1
+          table.table
+            th
+              |  Ticket Type
+            th
+              |  Unit Price
+            th
+              |  Quantity
+            - @event.ticket_types.each do |type|
```

Comment:
Good style `f.object` instead of `@event`, also use the same style for `ff` in the new approach we discussed.